Formula/znapzend.rb

```diff
@@ -20,6 +21,49 @@ def install
     system "make", "install"
   end
 
+  def post_install
+    (var/"log/#{name}").mkpath
+    (var/"run/#{name}").mkpath
+  end
+
+  plist_options(:startup => true, :manual => "znapzend --daemonize")
+
+  def plist; <<~EOS
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+          <key>PATH</key>
+          <string>/usr/bin:/bin:/usr/sbin:/sbin:#{HOMEBREW_PREFIX}/bin</string>
```

Comment:
does this still work when HOMEBREW_PREFIX is not /usr/local? i.e. does znapzend know where to find `zpool` if it's not in PATH here?

Reply:
Good point. It doesn’t unless `zpool` happens to live in `HOMEBREW_PREFIX`, too, which is unlikely; I only just realized the `openzfs` cask has `/usr/local/bin` hard-coded.

I’ll insert `/usr/local/bin`.